### DISCUSSION TITLE
Rotation of mesh plots?

### DISCUSSION THREAD

**Discussion by mr483** [NatLabRockies/floris#348] [Q&A] [answered]
Is there a simple solution to rotating the mesh plots used by floris.tools.visualization.visualize_cut_plane()? I looked for some techniques to rotate the QuadMesh object it returns but have not found anything useful. In general, I would like the wake visualization to always have the same orientation, regardless of the wind direction applied during the simulation. I recognize that the simulation is only utilizing the grid area aligned with the wind direction, and I'm fine with this - but would like the output plots to have a consistent orientation such that I can quickly review plots under various conditions. 

Thanks in advance for any advice!

**Comment by paulf81** [comment, marked as answer]
This is a good suggestion, I have opened Issue #357 to address it, please do feel free to comment there if you think this will address this comment.  Thank you!
